cmake/onnxruntime_mlas.cmake

```diff
@@ -99,6 +100,10 @@ function(setup_mlas_source_for_windows)
         ${MLAS_SRC_DIR}/halfgemm_kernel_neon_fp16.cpp
       )
 
+      setup_kleidiai()
```

Comment:
It means that kleidiai will be a new dependency for all ONNX Runtime build configs.  For such changes the onnx runtime team needs to hold an internal discussion with the leadership of this project.